resources/validate_builds.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+# validated if created build is debug build 
+cd $APK_PATH
+
+APK_NAME=""
+EXPECTED_FLAG='application-debuggable'
+
+for file in *.apk
+do
+  APK_NAME=$file  
+done
```

Comment:
Are you expecting multiple apk files? If not, why not just hard code the name of the apk?

Reply:
Yap, we may expect multiple apks